recipes/flake8-noqa/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "flake8-noqa" %}
+{% set version = "1.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flake8-noqa-{{ version }}.tar.gz
+  sha256: 02f6a10d0e59c0d50682a8c2799bf788dc7ae0eed27a4400ea66ea18ab42397c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - flake8 >=3.8.0,<5.0
+    - python >=3.6
+    - typing-extensions <5.0,>=3.7.4.2
+
+test:
+  imports:
+    - flake8_noqa
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/plinss/flake8-noqa/
+  summary: Flake8 noqa comment validation
+  license: LGPL-3.0-only
+  license_file: COPYING.LESSER
```

```suggestion
  license_file: 
    - COPYING.LESSER
    - LICENSE
```
Both should be included.